common/motion_utils/include/motion_utils/trajectory/trajectory.hpp

```diff
@@ -703,11 +703,13 @@ inline boost::optional<geometry_msgs::msg::Point> calcLongitudinalOffsetPoint(
  * @param points points of trajectory, path, ...
  * @param src_idx index of source point
  * @param offset length of offset from source point
+ * @param use_slerp_for_orientation set orientation by spherical interpolation if true
  * @return offset pose
  */
 template <class T>
 inline boost::optional<geometry_msgs::msg::Pose> calcLongitudinalOffsetPose(
-  const T & points, const size_t src_idx, const double offset, const bool throw_exception = false)
+  const T & points, const size_t src_idx, const double offset,
+  const bool use_slerp_for_orientation = false, const bool throw_exception = false)
```

Comment:
[imo]
I think it is better to use common flag name for readability. What do you think?

https://github.com/autowarefoundation/autoware.universe/blob/fa25a2fe4625bf345a97a27f66c5d30d1ca9854a/common/tier4_autoware_utils/include/tier4_autoware_utils/geometry/geometry.hpp#L667-L670

Reply:
I think so too. :+1: 
fixed.
https://github.com/autowarefoundation/autoware.universe/pull/1575/commits/f7bf15ab1a04a34709a0282a4be2f855e2cf1770

Reply:
I confirmed unit test has passed after this change.